storybook/stories/ui-charts/manchetteWithSpaceTimeChart/simple.stories.tsx

```diff
@@ -6,7 +6,7 @@ import type { Meta } from '@storybook/react';
 import { SAMPLE_WAYPOINTS, SAMPLE_PATHS_DATA } from './assets/sampleData';
 
 const meta: Meta<typeof ManchetteWithSpaceTimeChart> = {
-  title: 'Manchette with SpaceTimeChart/simple',
+  title: 'Manchette with SpaceTimeChart/ManchetteWithSpaceTimeChart',
```

Comment:
we have a story ManchetteWithSpaceTimeChart and useManchetteWithSpaceTimeChart
do we need both ?
can we have a more explicit name here too ?

Reply:
The two stories show how to use the two different APIs `ManchetteWithSpaceTimeChart` (the component) and  `ManchetteWithSpaceTimeChart` (the hook). I don't see the issues with these stories being there, nor with there names, actually.

It's still the same debate I think: The name of these stories is based on their code content, not on what they show on screen.

Reply:
hm, okay makes sense to show these two ways to use. it’s just that the names aren’t really helpful. we already know it’s manchette + space time since it’s the name of the section.

![Screenshot_20250326_203839](https://github.com/user-attachments/assets/762b7624-7c24-48d5-a1e5-4c62606ca280)

Reply:
perhaps just name it `component api` and `hook api`.

Reply:
OK, I like these better as well, good solution. I just pushed a fix.